content/docs/0.8.1/snapshots-and-backups/backup-and-restore/set-backup-target.md

```diff
@@ -122,6 +130,6 @@ The target URL should look like this:
 nfs://longhorn-test-nfs-svc.default:/opt/backupstore
 ```
 
-You can find an example NFS backupstore for testing purpose [here](https://github.com/longhorn/longhorn/blob/master/deploy/backupstores/nfs-backupstore.yaml). 
+You can find an example NFS backupstore for testing purpose [here](https://github.com/longhorn/longhorn/blob/master/deploy/backupstores/nfs-backupstore.yaml).
 
 **Result:** Longhorn can store backups in S3. To create a backup, see [this section.](../create-a-backup)
```

Comment:
@catherineluse is wrong and should probably read in NFS?

Reply:
Yeah, here should be `NFS` instead of `S3`.